### PR TITLE
Extract and test getFontWeights

### LINKED ISSUE #1381: Multi-selection with 2 different font families crashes the editor
## Bug Description

Looks like when two elements with different font families are selected, the editor crashes.

## Expected Behaviour
Editor should not crash.

## Steps to Reproduce
1. Add two text elements.
2. Assign non-default font to one of the elements, e,g. Mukta Mahee
3. Lasso select the elements
4. See the editor crashing.

## Screenshots
crashing
Screenshot 2020-04-29 at 22 45 59

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by swissspidy**
## Summary

Adds fallbacks that were previously there.

## Relevant Technical Choices

Extracted the method into its own file to make it easier to test.

---

Fixes #1381

### CODE REVIEW

File: assets/src/edit-story/components/panels/textStyle/getFontWeights.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+/**
+ * Returns a list of human-readable font weights.
+ *
+ * @param {Object} [font] Font object.
+ * @param {Array<number>} font.weights Font weights.
+ * @return {Array<Object<{name: string, value: string}>>}} Font weights list.
+ */
+function getFontWeights(font) {
+  const fontWeightNames = {
```
Comment (dvoytenko) [inline assets/src/edit-story/components/panels/textStyle/getFontWeights.js]: Let's make this a top-level const?

**Review by dvoytenko — DISMISSED** [review]

File: assets/src/edit-story/components/panels/textStyle/getFontWeights.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ * Returns a list of human-readable font weights.
+ *
+ * @param {Object} [font] Font object.
+ * @param {Array<number>} font.weights Font weights.
+ * @return {Array<Object<{name: string, value: string}>>}} Font weights list.
+ */
+function getFontWeights(font) {
+  const defaultFontWeights = [{ name: fontWeightNames[400], value: '400' }];
+
+  if (!font) {
```
Comment (miina) [inline assets/src/edit-story/components/panels/textStyle/getFontWeights.js]: Mb just `if (!font?.weights) return defaultWeights`?

**Review by miina — APPROVED** [review]
Left one optional comment, otherwise LGTM, works, too.

File: assets/src/edit-story/components/panels/textStyle/getFontWeights.js (hunk 2 above)
Comment (swissspidy) [reply assets/src/edit-story/components/panels/textStyle/getFontWeights.js]: Guess it was a bit late to think about optional chaining... Makes sense to add 👍

Comment (swissspidy) [reply assets/src/edit-story/components/panels/textStyle/getFontWeights.js]: Fixed!

**Pull request merged**